corehq/apps/userreports/models.py

```diff
@@ -462,6 +464,16 @@ def all(cls):
                     doc['_id'] = cls.get_doc_id(domain, doc['table_id'])
                     yield DataSourceConfiguration.wrap(doc)
 
+        for provider_path in settings.STATIC_DATA_SOURCES:
+            try:
+                provider_fn = to_function(provider_path, failhard=True)
+            except:
```

Comment:
I had thought that I wouldn't want to prevent django running if this failed but I think you're right that it should fail hard to indicate bad configuration